.github/workflows/pullrequest-trusted.yml

```diff
@@ -0,0 +1,39 @@
+
+name: Pull Request Update
+
+concurrency: 
+  group: pullrequest-${{ github.head_ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request_target:
+    branches: [ 'qaworkflow' ]
+    types: [ 'labeled' ]
+
+# To test workflow updates you need to work in a branch directly on viamrobotics/viam-cartographer
+# and tag your working branch instead of @main in any viamrobotics/viam-cartographer "uses" below.
+# Don't forget to tag back to @main before merge.
+
+jobs:
+  test:
+    if: (github.event.label.name == 'safe to test' || github.event.label.name == 'appimage')
```

Comment:
Shouldn't this be 
```suggestion
    if: (github.event.label.name == 'safe to test' && github.event.label.name == 'appimage')
```

Reply:
No, because there are plenty of cases where you don't need/want to build an appimage, but you DO want to test.

Reply:
What about the instance in which a PR is labeled appimage but not safe to test? Should this be able to trigger test since they are required by the appimage job?

Reply:
If you demand something by adding a label, you should get it and not have to add TWO things to get one outcome. You made your choice. :-)

(Also, we do it this way in RDK, so consistency is good.)